src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -1,29 +1,49 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { ResultSectionDeps } from 'common/components/cards/result-section';
+import { VisualizationToggle } from 'common/components/visualization-toggle';
 import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
 import { NamedFC } from 'common/react/named-fc';
 import { FeatureFlagStoreData } from 'common/types/store-data/feature-flag-store-data';
+import { TabStoreData } from 'common/types/store-data/tab-store-data';
 import { VisualizationScanResultData } from 'common/types/store-data/visualization-scan-result-data';
+import { VisualizationStoreData } from 'common/types/store-data/visualization-store-data';
 import { VisualizationType } from 'common/types/visualization-type';
 import { RequirementInstructions } from 'DetailsView/components/requirement-instructions';
 import * as styles from 'DetailsView/components/static-content-common.scss';
+import { StaticContentDetailsViewDeps } from 'DetailsView/components/static-content-details-view';
 import {
     TabStopsFailedInstanceSection,
     TabStopsFailedInstanceSectionDeps,
 } from 'DetailsView/components/tab-stops-failed-instance-section';
+import {
+    TabStopsRequirementsTable,
+    TabStopsRequirementsTableDeps,
+} from 'DetailsView/components/tab-stops/tab-stops-requirements-table';
+import { DetailsViewToggleClickHandlerFactory } from 'DetailsView/handlers/details-view-toggle-click-handler-factory';
 import { createFastPassProviderWithFeatureFlags } from 'fast-pass/fast-pass-provider';
 import * as React from 'react';
+import { ContentLink, ContentLinkDeps } from 'views/content/content-link';
+import { ContentReference } from 'views/content/content-page';
 import * as Markup from '../../assessments/markup';
 
-export type AdhocTabStopsTestViewDeps = TabStopsFailedInstanceSectionDeps;
+export type AdhocTabStopsTestViewDeps = TabStopsRequirementsTableDeps &
+    TabStopsFailedInstanceSectionDeps &
+    ResultSectionDeps &
+    StaticContentDetailsViewDeps &
```

Comment:
Neither ResultSection or StaticContentDetailsView are child components here so we should remove the deps for them. If we need something in this component we should be adding it here explicitly.